templates/nextjs-boilerplate-template/package.json

```diff
@@ -52,6 +53,7 @@
     "typescript": "^5.2.2",
     "tree-kill": "1.2.2",
     "dotenv": "16.3.1",
-    "vercel": "^35.2.4"
+    "vercel": "^35.2.4",
+    "webpack": "^5.95.0"
```

Comment:
och... is that a must?

Reply:
i think so https://ducanh-next-pwa.vercel.app/docs/next-pwa/getting-started

Reply:
actuallly i removed it and it still works, remove now